telegram_bot/bot.py

```
@@ -0,0 +1,434 @@
+        await _reply(update, f"Failed to fetch trade history: {exc}")
+
+
+# ---------------------------------------------------------------------------
+# Trading commands — IOC
+# ---------------------------------------------------------------------------
+
+
+def _parse_order_args(args: list[str]) -> tuple[str, str, str]:
+    """Parse and validate <SYMBOL> <QTY> <PRICE> args. Returns (symbol, qty, price)."""
+    if len(args) < 3:
+        raise ValueError("Usage: `<SYMBOL> <QTY> <PRICE>`")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)